examples/capacities_and_potentials.ipynb

```diff
@@ -0,0 +1,1282 @@
+      "text/plain": [
+       "<Figure size 1000x500 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# get the OCP functions\n",
+    "U_n = tea_class.parameter_values.get(\"Negative electrode OCP [V]\")\n",
+    "U_p = tea_class.parameter_values.get(\"Positive electrode OCP [V]\")\n",
+    "\n",
+    "# get the stoichiometries at 0% and 100% SoC\n",
+    "x0 = tea_class.stack_energy.get(\"Negative electrode stoichiometry at 0% SoC\")\n",
+    "x100 = tea_class.stack_energy.get(\"Negative electrode stoichiometry at 100% SoC\")\n",
+    "y0 = tea_class.stack_energy.get(\"Positive electrode stoichiometry at 0% SoC\")\n",
+    "y100 = tea_class.stack_energy.get(\"Positive electrode stoichiometry at 100% SoC\")\n",
+    "\n",
+    "# create vectors of stoichiometries\n",
+    "soc = pybamm.linspace(0, 1)\n",
+    "x = x0 + (x100 - x0) * soc\n",
+    "y = y100 + (y0 - y100) * soc\n",
```

Comment:
check how you've defined this. right now at soc=1 you have that y = y0, not y100. I think it should be 
```
            x = x_0 + soc * (x_100 - x_0)
            y = y_0 - soc * (y_0 - y_100)
```
but also check how you have defined x_0, x_100, y_0, y_100. these should be defined wrt to cell level SOC